### ISSUE TITLE
[Enhancement/Feature Request] Add tape transport controls to main toolbar

### ISSUE THREAD

**Issue by wibblemonkey** [ArjunNair/Zero-Emulator#51]
Would it be possible to duplicate the tape transport controls on the main window toolbar? It's quite inconvenient to always have to open the Tape Browser window to start a tape when edge/auto loading is turned off.

**Comment by ArjunNair** [comment]
Yeah. It's a good idea. It reduces the screen estate a little bit, but probably worth it.
